src/StarRating/StarRating.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Text, StyleSheet } from 'react-native'
+import StarButton from './StarButton'
+
+const range = [0, 1, 2, 3, 4]
+
+export default class StarRating extends Component {
+  static propTypes = {
+    size: PropTypes.number,
+    count: PropTypes.number,
```

Comment:
I think count may be not only number. It can be a string too, if:
```javascript
<Component count="10">
```

Maybe add explicit transform into Number(str) ?